numpy/_core/fromnumeric.py

```diff
@@ -918,6 +920,13 @@ def argpartition(a, kth, axis=-1, kind='introselect', order=None):
 
     Notes
     -----
+    The returned indices are not guaranteed to be sorted according to
+    the values. Furthermore, the default selection algorithm ``introselect``
+    is unstable, and hence the returned indices are not guaranteed
+    to be the earliest/latest occurrence of the element.
+
+    The sort order of ``np.nan`` is bigger than ``np.inf``.
+
     See `partition` for notes on the different selection algorithms.
```

Comment:
I don't understand this comment. Aren't we in the `partition` function docstring? Does this docstring show up in a different function as well?

Reply:
The sort order is defined in the docstring for `np.sort`, but since `np.partition` uses a different implementation, I thought it would be helpful to also define the sort order of `np.partition`.

I've removed notes regarding the sort order from `np.argpartition` and instead linked it to the docs for `np.partition`, this follows what `np.sort` and `np.argsort` did.